var/spack/repos/builtin/packages/meep/package.py

```diff
@@ -72,6 +86,11 @@ def configure_args(self):
         else:
             config_args.append('--without-hdf5')
 
+        # make meep-python fail to compile
+        # config_args.append('--without-scheme')
+        # --with-scheme need maintainer mode...
```

Comment:
Maybe something like this:
```suggestion
        # meep-python + scheme (implicit) fails to build without maintainer mode
```

Reply:
Also, aren't we maybe missing a dependency on `python`?

Reply:
Yes that is correct, I added the missing dependencies in 59546ee